src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/MediaPlayerElement/MediaPlayerElementTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+using Uno.UITests.Helpers;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.MediaPlayerElementTests;
+
+[TestFixture]
+class MediaPlayerElementTests : SampleControlUITestBase
+{
+	[Test]
+	[AutoRetry]
+	public void MediaPlayerElement_PlayPauseStopTest()
+	{
+		Run("UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Original");
+
+		_app.WaitForElement(_app.Marked("MpeOriginal"));
+
+		TakeScreenshot("Initial MpeOriginal");
+
+		// step 1: Test Auto Play
+		var itemQuery = _app.Marked("TimeElapsedElement");
+		_app.WaitForText(itemQuery, "0:00:04");
```

Comment:
Let's not wait on explicit time, let's use the actual value parsed to a TimeSpan and a greater than comparison